src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -164,7 +164,7 @@ public async Task it_can_load_assembly_references_using_r_directive_with_relativ
         public async Task it_can_load_script_files_using_load_directive_with_relative_path(Language language, bool changeWorkingDirectory)
         {
             var srcDir = DirectoryUtility.GetPathToSrcDirectory();
-            var pathToScripts = Path.Combine(srcDir, @"Microsoft.DotNet.Interactive.Tests\");
+            var pathToScripts = Path.Combine(srcDir, @"Microsoft.DotNet.Interactive.Tests"+Path.DirectorySeparatorChar);
```

Comment:
do we need the trailing `'`'` at all?

Reply:
no, you are right.
Probably it would be better to put a forward-slash in these 2 lines before "RelativeLoadingSample":

https://github.com/dotnet/interactive/blob/5a5a652d6f67e1ed1e05eed28061c4216a188b0d/src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs#L188-L189